cuebot/build.gradle

```diff
@@ -35,32 +37,31 @@ repositories {
 }
 
 dependencies {
-    compile group: 'com.google.code.gson', name: 'gson', version: '2.3.1'
-    compile group: 'com.google.guava', name: 'guava', version: '12.0.1'
+    compile group: 'com.google.code.gson', name: 'gson', version: '2.8.6'
+    compile group: 'com.google.guava', name: 'guava', version: '26.0-android'
     compile group: 'com.sun.mail', name: 'mailapi', version: '1.5.4'
     compile group: 'commons-lang', name: 'commons-lang', version: '2.6'
     compile group: 'io.grpc', name: 'grpc-all', version: '1.14.0'
     compile group: 'org.apache.activemq', name: 'activemq-pool', version: activemqVersion
     compile group: 'org.apache.velocity', name: 'velocity', version: '1.7'
     compile group: 'org.jdom', name: 'jdom', version: '1.1.3'
-    compile group: 'org.springframework.boot', name: 'spring-boot-starter-jdbc', version: springBootVersion
-    compile group: 'org.springframework.boot', name: 'spring-boot-starter-log4j', version: springBootVersion
-    compile group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: springBootVersion
-    compile group: 'org.springframework', name: 'spring-context-support', version: springVersion
-    compile group: 'org.springframework', name: 'spring-jms', version: springVersion
+    compile group: 'org.springframework.boot', name: 'spring-boot-starter-jdbc'
```

Comment:
How do the versions get selected here? Could you add a note?

Reply:
I've added a comment to address this. The spring.dependency-management plugin will locate the appropriate tested version of dependencies based on the specified spring boot version. So you only need to specify the spring boot version and everything else will follow along.
https://docs.spring.io/spring-boot/docs/current/gradle-plugin/reference/html/#managing-dependencies

Reply:
Ahh that's right, thanks!